libs/navigation/src/drivers/magento/queries/fragments/category-node.spec.ts

```diff
@@ -195,7 +195,23 @@ describe('Navigation | Driver | Magento | getCategoryNodeFragment', () => {
         data: depth3NavigationTree
       })
     });
-  });
+	});
+	
+	it('should return the expected fragment', () => {
```

Comment:
The spec message should be `should not use nested fragments` since that is specifically the behavior that we are targeting to workaround the bug.

Reply:
changed